### DISCUSSION TITLE
PHP 8: News rating funktionslos (Variablen wie MID werden nicht aus Template geladen)

### DISCUSSION THREAD

**Discussion by niclasd** [scheb/open-apexx#56] [General]
Hi Leute,

vielleicht hat ja einer das Problem bereits erkannt und einen Lösungsansatz für mich? :)

Die Bewertungsfunktion unter den News funktioniert mit dem Wechsel auf PHP 8 nicht mehr.

Template
`<input type="submit" name="sendrate" value="{LANG[SUBMITRATE]}" class="button" style="font-size:10px;" />
<input type="hidden" name="module" value="{MODULE}" />
<input type="hidden" name="mid" value="{MID}" />
`
HTML im Newsbeitrag
`<input type="submit" name="sendrate" value="Vote!" class="button" style="font-size:10px;" />
<input type="hidden" name="module" value="" />
<input type="hidden" name="mid" value="" />`

Bin für jede Lösung dankbar.

**Comment by bigfraggle** [comment]
Ich nutze die Funktion nicht, daher habe ich jetzt keine Lösung parat. Aber du solltest mal schauen, dass du in der Config die Fehleranzeige aktivierst. Mit einer konkreten Fehlermeldung sollte es dann auch einfacher sein, das Problem zu lokalisieren.

**Comment by niclasd** [comment]
> Ich nutze die Funktion nicht, daher habe ich jetzt keine Lösung parat. Aber du solltest mal schauen, dass du in der Config die Fehleranzeige aktivierst. Mit einer konkreten Fehlermeldung sollte es dann auch einfacher sein, das Problem zu lokalisieren.

Guter Hinweis, werde ich probieren. Danke. 
Es scheint auch andere Funktionen zu betreffen, die irgendwie in Inhalte integriert sind - so werden die Thumbnails einer Galerie problemlos angezeigt, allerdings werden die notwendigen Informationen nicht mehr ins Pooup übergeben (also die Einzelbildansicht, die sich in einem neuen Fenster öffnet)

**Comment by niclasd** [comment]
> Ich nutze die Funktion nicht, daher habe ich jetzt keine Lösung parat. Aber du solltest mal schauen, dass du in der Config die Fehleranzeige aktivierst. Mit einer konkreten Fehlermeldung sollte es dann auch einfacher sein, das Problem zu lokalisieren.

Guter Hinweis, werde ich probieren. Danke. 
Es scheint auch andere Funktionen zu betreffen, die irgendwie in Inhalte integriert sind - so werden die Thumbnails einer Galerie problemlos angezeigt, allerdings werden die notwendigen Informationen nicht mehr in die Einzelbildansicht übergeben, die sich in einem neuen Fenster öffnet.

**Comment by bigfraggle** [comment]
Bei den geschlossenen Beiträgen gibt es ansonsten ja bereits ein paar Sachen zu PHP8. Es ist nicht unwahrscheinlich, dass hier das Problem ähnlich ist.
